strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/cron/jobs/ClearTrashCronJobFromMaven2RepositoryTestIT.java

```diff
@@ -49,6 +50,7 @@
     @Test
     @ExtendWith({ RepositoryManagementTestExecutionListener.class,
                   ArtifactManagementTestExecutionListener.class })
+    @Disabled
```

Comment:
Is there a follow-up task for sorting this test out? If no, could you please create one? Thanks!

Reply:
#1973
It seems I've already mentioned this issue above :)

Reply:
Okay, thank for clarifying!